doc/Makefile

```
@@ -78,3 +78,8 @@ linkcheck:
 
 linkcheck-grep:
 	@! grep -h "^.*:.*: \[\(\(local\)\|\(broken\)\)\]" _build/linkcheck/output.txt
+
+# Spin up a local http server to view the rendered documentation.
+# This is required for interactive examples to work.
+serve-html:
+	python -m http.server 0 --directory "$(BUILDDIR)"/html
```

I know I originally suggest port 0 to avoid the default port that is overused, but after trying this a few times I think we should fix the port to something like `15000`:

```suggestion
	python -m http.server 15000 --directory "$(BUILDDIR)"/HTML
```